src/ImageService.Data/ImageNewsRepository.cs

```diff
@@ -39,7 +39,7 @@ public bool Remove(List<Guid> imageIds)
       foreach (Guid imageId in imageIds)
       {
         _provider.ExecuteRawSql($@"DELETE FROM {DbImageNews.TableName} WHERE Id = '{imageId}' OR ParentId = '{imageId}' OR
-            Id IN (SELECT ParentId FROM ImagesProjects WHERE Id = '{imageId}' AND ParentId IS NOT NULL);");
+          Id IN (SELECT ParentId FROM ImagesProjects WHERE Id = '{imageId}' AND ParentId IS NOT NULL);");
```

Comment:
change ImagesProjects to {DbImageNews.TableName}

Reply:
done